docs/InstallGuide.md

```diff
@@ -1,424 +1,358 @@
-- **port** : The port number for connecting with mysqld. If necessary.
-- **user** : The user name for connecting.
-- **password** : The password for connecting.
+### Setup Database
 
-Creating Database
------------------
+Edit `/etc/openvnet/common.conf` if necessary. The sample environment uses the default settings.
 
-Before creating the database, you need to launch the mysql server.
+Launch mysql server.
 
     # service mysqld start
 
-To automatically launch the mysql server, execute the following command.
+To automatically launch the mysql server at boot, execute the following command.
 
     # chkconfig mysqld on
 
-Create Database
-
-    # mysqladmin -uroot create vnet
-    # cd /opt/axsh/openvnet/vnet
-    # bundle exec rake db:init
-
-
-Start the OpenVNet services
----------------------------
-
-    # initctl start vnet-vnmgr
-    # initctl start vnet-webapi
-    # initctl start vnet-vna
-
-OpenVNet writes its logs in the /var/log/openvnet directory. If there's a problem starting any of the services, you can find its log files there.
-
-
+Set `PATH` environment variable as following since the OpenVNet uses its own ruby binary.
 
----------------------------------------------------------------------
-
-# Let's try 1Box OpenVNet#
-
-This section explains how to install OpenVNet on one node. This is a great way to try it out.
-![1BOX](OpenVNet_1Box_environment.gif)
+```
+# PATH=/opt/axsh/openvnet/ruby/bin:${PATH}
+```
 
-Install the following operation system and kernel.
+Create database
 
-+ RHEL (or Centos) 6.4
-+ Kernel 2.6.32-358.6.2.el6.x86_64
+```
+# cd /opt/axsh/openvnet/vnet
+# bundle exec rake db:create
+# bundle exec rake db:init
+```
 
+Start vnmgr and webapi.
 
-Installation and Setup of OpenVNet
------------------------------------
+```
+# initctl start vnet-vnmgr
+# initctl start vnet-webapi
+```
 
-### Network interfaces setup
+Subsequent database records are required for the sake of the sample environment.
 
-We're going to enslave eth0 to openvswitch so OpenVNet can use it. You will no longer be able to use eth0 to ssh into the node or access the Internet. Prepare another interface for that if you need to.
+#### Datapath
 
-    # vi /etc/sysconfig/network-scripts/ifcfg-eth0
-    DEVICE=eth0
-    DEVICETYPE=ovs
-    TYPE=OVSPort
-    OVS_BRIDGE=br0
-    BOOTPROTO=none
-    ONBOOT=yes
-    HOTPLUG=no
-    #
-    # vi /etc/sysconfig/network-scripts/ifcfg-br0
-    DEVICE=br0
-    DEVICETYPE=ovs
-    TYPE=OVSBridge
-    ONBOOT=yes
-    BOOTPROTO=static
-    IPADDR=172.16.20.41
-    NETMASK=255.255.255.0
-    HOTPLUG=no
-    OVS_EXTRA="
-     set bridge     ${DEVICE} protocols=OpenFlow10,OpenFlow12,OpenFlow13 --
-     set bridge     ${DEVICE} other_config:disable-in-band=true --
-     set bridge     ${DEVICE} other-config:datapath-id=00004e6d2b508f4c --
-     set bridge     ${DEVICE} other-config:hwaddr=02:01:00:00:00:01 --
-     set-fail-mode  ${DEVICE} standalone --
-     set-controller ${DEVICE} tcp:127.0.0.1:6633
-    "
```

Comment:
This is the first time we're using vnctl so a quick sentence to explain that it is a web api client would be a good idea.

Also I think it's not completely clear what we're adding here. The reader might think like "What? I've already created a datapath earlier. Why do I do it again now?" We should explain the we are now telling OpenVNet to use the datapath that we have created earlier.